libkrun/libkruntime.c

```diff
@@ -30,7 +30,7 @@
 #define ACTUAL_CLOCK_MONOTONIC    CLOCK_MONOTONIC
 #endif
 
-#if defined(__linux__) && !defined(NO_MSRS)
```

Comment:
Do I see it right that throughout the PR `NO_MSRS` was renamed to `MSRS`, but the surrounding logic wasn't negated? That has to be a mistake?

For example, in this case I think you mean:
```
#if defined(__linux__) && defined(MSRS)
```

and not:
```
#if defined(__linux__) && !defined(MSRS)
```

(remove the leading `!` negation after the `&&`)

Reply:
Eek, you're quite right! Fixed in bf4daf9.

Reply:
And as a bonus, make the comments correct too in 72cda92.